CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## v1.0.0-beta.0
+
+ENHANCEMENTS:
+
+- `resource/auth0_action`: Add support for GA version of node 18 within `runtime` field ([#709](https://github.com/auth0/terraform-provider-auth0/pull/709), [#722](https://github.com/auth0/terraform-provider-auth0/pull/722))
+- `resource/auth0_trigger_actions`: Add `password-reset-post-challenge` to available trigger bindings ([#726](https://github.com/auth0/terraform-provider-auth0/pull/726))
+- `resource/auth0_trigger_action`: Add `password-reset-post-challenge` to available trigger bindings ([#746](https://github.com/auth0/terraform-provider-auth0/pull/746))
+- `resource/auth0_*`: Improve how we check for 404 errors ([#727](https://github.com/auth0/terraform-provider-auth0/pull/727))
+- `resource/auth0_client`: Add validation to prevent empty `mobile block ([#742](https://github.com/auth0/terraform-provider-auth0/pull/742))
```

Comment:
```suggestion
- `resource/auth0_client`: Add validation to prevent empty `mobile` block ([#742](https://github.com/auth0/terraform-provider-auth0/pull/742))
```